k8s/overlays/azure/dev/postgresql-deployment.yaml

```diff
@@ -217,15 +226,21 @@ spec:
               cpu: "500m"
               memory: 1Gi
           volumeMounts:
+            - name: postgresql-data
+              mountPath: /bitnami/postgresql
+#              Temporary, used only for moving to properly using persistence, to delete afterwards
```

Comment:
It was not clear for me on first reading that you were referring to `postgres-data` and not `postgresql-data`.
Maybe move this comment at the end of the line below.